docs/UserGuide.adoc

```diff
@@ -295,6 +295,56 @@ To end step-by-step instruction of cook-mode. +
 Format: +
 `end`
 
+=== Health-planner Command
+
+==== Add plan: `add`
+
+Adds a health plan under the user.
+Format: `add GOAL DURATION`
```

Comment:
Please leave a new line after "Format:". 
e.g. :
Format: +

Do specific more information for GOAL and DURATION. e.g. the range of values